src/os/bluestore/bluestore_tool.cc

```diff
@@ -594,6 +603,67 @@ int main(int argc, char **argv)
       exit(EXIT_FAILURE);
     }
   }
+  else if (action == "bluefs-import") {
```

Comment:
i'd suggest extract this command handling into a separated function. yeah, better off starting doing so now.